worlds/sc2/docs/contributors.md

```diff
@@ -0,0 +1,42 @@
+# Contributors
+Contibutors are listed with preferred or Discord names first, with github usernames prepended with an `@`
+
+## Update 2024.0
+### Code Changes

```

Comment:
technically I contributed code back in the Condor era, but it's mostly cleanup and a bit of the mission launcher UI.

Reply:
Updated